AtbIntegrationTests/AtbIntegrationTests.swift

```diff
@@ -0,0 +1,232 @@
+//
+//  AtbIntegrationTests.swift
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+import Swifter
+
+class AtbIntegrationTests: XCTestCase {
+
+    struct Constants {
+        static let initialAtb = "v100-1"
+        static let retentionAtb = "v102-7"
+        static let devmode = "test"
+        static let atbParam = "atb"
+        static let setAtbParam = "set_atb"
+    }
+    
+    let app = XCUIApplication()
+    let server = HttpServer()
+    var searchRequests = [HttpRequest]()
+    var extiRequests = [HttpRequest]()
+    var atbRequests = [HttpRequest]()
+    var atbToSet = Constants.initialAtb
+    
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        
+        atbToSet = Constants.initialAtb
+        
+        app.launchEnvironment = [
+            "BASE_URL": "http://localhost:8080",
+            "BASE_PIXEL_URL": "http://localhost:8080"
```

Comment:
Is this used?

Reply:
not explicitly, but it prevents pixels being fired in to the real system for this test scenario